### DISCUSSION TITLE
how to make own text det dataset

### DISCUSSION THREAD

**Discussion by laopeng111** [open-mmlab/mmocr#1759] [Q&A]
please help!
I want to make own text detection dataset.But i can't make success from mmocr/data_converter.
my text det  dataset groundtruth  displayed in pic.

![捕获](https://user-images.githubusercontent.com/127070252/223056383-789ac9e6-4056-435e-8064-76963d919be2.PNG)

**Comment by gaotongxiao** [comment]
If you are using mmocr 1.x (`1.x` or `dev-1.x` branch), just write a script and convert it to the following format:
https://mmocr.readthedocs.io/en/dev-1.x/migration/dataset.html#introduction-of-the-new-format

**Reply by laopeng111** [reply]
Thanks for your answer.But i don't have old version gt file ,how to use write scipte for example(**python tools/dataset_converters/textdet/data_migrator.py ${IN_PATH} ${OUT_PATH}**) to get new gt file.

**Reply by gaotongxiao** [reply]
That script is prepared for those who want to migrate from MMOCR 0.x, which is not applicable to your case. You'll have to write a custom script and convert your annotation to the new data format.

**Reply by laopeng111** [reply]
Thanks so much.I'll try to write custom sctipt.
